README.md

```diff
@@ -19,13 +19,13 @@ $ go get -u github.com/decred/base58
 
 ## Examples
 
-* [Decode Example](http://godoc.org/github.com/decred/base58#example-Decode)  
+* [Decode Example](https://godoc.org/github.com/decred/base58#example-Decode)
```

Comment:
Removing the spaces at the end will mess up the formatting.  They should be left.

Reply:
reverted